### DISCUSSION TITLE
How to make the slot button follow the animation

### DISCUSSION THREAD

**Discussion by bamiendev** [google/model-viewer#4649] [Q&A]
https://github.com/google/model-viewer/assets/116520283/0a8c5369-893f-40db-91c8-ed5d1b96e337

I have a video describing animation
 How to make the slot button follow the animation

**Comment by bamiendev** [comment]
I have an idea using functions updateHotspot() to update position and normal of slot button, but how to know their position when running animation.

**Reply by elalish** [reply]
We have an example for exactly this: https://modelviewer.dev/examples/annotations/index.html#animatedHotspots. If you use our editor to place hotspots, it automatically writes them in this form to work with animated models.
